polychrom/hdf5_format.py

```diff
@@ -96,13 +96,24 @@ def load_hdf5_file(fname):
             
 class hdf5Reporter(object):
```

Comment:
Any reason for camelCase instead of CapitalCase on a class?

Reply:
Not really. Just forgot. Should we generally use understores instead of camel case? I think I'm ready to switch personally.